wtr-utils.js

```diff
@@ -9,7 +9,7 @@ const { visualRegressionPlugin } = require('@web/test-runner-visual-regression/p
 
 const HIDDEN_WARNINGS = [
   '<vaadin-crud> Unable to autoconfigure form because the data structure is unknown. Either specify `include` or ensure at least one item is available beforehand.',
-  'The <vaadin-grid> needs the total number of items in order to display rows. Set the total number of items to the `size` property, or provide the total number of items in the second argument of the `dataProvider`’s `callback` call.',
+  'The <vaadin-grid> needs the total number of items in order to display rows, which you can specify either by setting the `size` property, or by providing it to the second argument of the `dataProvider` function `callback` call.',
```

Comment:
Rephrased the warning to not repeat `the total number of items` 3 times, and updated this check accordingly (some tests might fail with this warning in tests where we don't stub `console.warn()` and it's OK).